### DISCUSSION TITLE
Effect of values for "weight"?

### DISCUSSION THREAD

**Discussion by gabrielwhite** [searxng/searxng#2662] [Q&A] [answered]
I'm trying to work out how to weight the search engines in the settings.yaml.

I can see most values are set at 1 (default?), others at 2, and a couple of others at 100.

Is the weighting system using a range of 1 to 100? If I have search engine A at 1, and B at 2, is that any different from having A at 1 and B at 100?

I searched the docs and here, and wasn't able to turn up anything.

Thanks!

**Comment by return42** [comment, marked as answer]
> Is the weighting system using a range of 1 to 100?

No, you can also use 0.5 to lower the result-score

> If I have search engine A at 1, and B at 2, is that any different from having A at 1 and B at 100?

Yes, results from B are ranked 100 times higher then those from A .. in a result page results from B are shown before results from a

https://github.com/searxng/searxng/blob/d52919cc3e90b588e252e41e73f06aea49097db2/searx/results.py#L131-L140
